src/libopenid.cpp

```diff
@@ -1203,7 +1204,7 @@ static irods::error _get_provider_config( std::string key, boost::any& cfg )
 
 
 /*
-    Looks for a server_config.json string corresponding to key, withing hte openid config section.
+    Looks for a server_config.json string corresponding to key, withing the openid config section.
```

Comment:
while you're in here...      please change `withing`->`within`

Reply:
I changed "within", but it seems that you need to squash this yourself, because write access to the repository is needed.